config/cli_config_example.yml

```diff
@@ -0,0 +1,14 @@
+---
+:ui:
+  :interactive: true
+  :per_page: 20
+  :history_file: ~/.hammer/history
+:watch_plain: false
+:reload_cache: false
+:log_dir: ~/.hammer/log
+:log_level: error
+:global_defaults:
+  :organization_id: 293
+  :organization_name: Redhat
+:user_defaults:
+  :organization_id: 2
```

Comment:
Please nest the options one level deeper so it doesn't mess with other config options. Then the fragile `_defaults` suffix is not needed and it can make human editing and even processing more netural.
```yaml
:defaults:
  :_global:
    :organization_id: 293
    :organization_name: Redhat
  :user:
    :organization_id: 2
```